src/simsopt/geo/surfacerzfourier.py

```diff
@@ -705,7 +774,7 @@ def fourier_transform_scalar(self, scalar, mpol=None, ntor=None, normalization=N
         if not stellsym:
             A_mnc[0, ntor] = np.sum(scalar) / (ntheta_grid * nphi_grid)
         if normalization is not None:
-            if isinstance(normalization, float):
+            if not isinstance(normalization, float):
```

Comment:
Good catch!

Reply:
I'd be much happier if the mistake weren't my own ;)